projects/1.19/assets/crimsonsteves-more-mobs/crimson_steves_mobs/lang/zh_cn.json

```
@@ -0,0 +1,133 @@
+{
+  "entity.crimson_steves_mobs.animated_temple": "§8复苏的神庙",
+  "entity.crimson_steves_mobs.big_snow_golem": "大型雪傀儡",
+  "entity.crimson_steves_mobs.chad_creeper_golem": "苦力怕傀儡斗士",
+  "entity.crimson_steves_mobs.chad_villager": "村民斗士",
+  "entity.crimson_steves_mobs.creeckin": "苦力鸡",
+  "entity.crimson_steves_mobs.creeper_golem": "苦力怕傀儡",
+  "entity.crimson_steves_mobs.creepig": "苦力猪",
+  "entity.crimson_steves_mobs.creesteve": "苦力史蒂夫",
+  "entity.crimson_steves_mobs.crude_redstone_golem": "粗糙的红石傀儡",
+  "entity.crimson_steves_mobs.crude_redstone_monstrosity": "§4粗糙的红石巨兽",
+  "entity.crimson_steves_mobs.cyborg_vindicator": "机械报复者",
+  "entity.crimson_steves_mobs.derpig": "Derpig",
+  "entity.crimson_steves_mobs.elite_ender_ravager": "精英末地掠夺兽",
+  "entity.crimson_steves_mobs.ender_ravager": "末地掠夺兽",
```

```suggestion
  "entity.crimson_steves_mobs.elite_ender_ravager": "精英末影劫掠兽",
  "entity.crimson_steves_mobs.ender_ravager": "末影劫掠兽",
```
https://minecraft.fandom.com/zh/wiki/%E5%8A%AB%E6%8E%A0%E5%85%BD